Reviewer/src/main/java/com/softserveinc/reviewer/dao/StatisticDaoMongoJackImpl.java

```diff
@@ -0,0 +1,68 @@
+package com.softserveinc.reviewer.dao;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.NoSuchElementException;
+
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+import com.mongodb.BasicDBObject;
+import com.mongodb.DB;
+import com.mongodb.DBCollection;
+import com.mongodb.DBObject;
+import com.mongodb.MongoClient;
+import com.softserveinc.reviewer.model.Statistic;
+import org.mongojack.DBCursor;
+import org.mongojack.DBQuery;
+import org.mongojack.JacksonDBCollection;
+
+public class StatisticDaoMongoJackImpl implements StatisticDao {
+    private static final String DATABASE_NAME = "test";
+    private static final String COLLECTION_NAME = "collection";
```

Comment:
That should be a part of application configuration, rather than class properties